src/session.rs

```diff
@@ -48,7 +294,7 @@ pub(crate) struct Task {
 }
 
 impl Client {
-    pub(crate) fn issue<C>(&self, cmd: C) -> impl Future<Output = Result<Json, error::CmdError>>
+    pub(crate) async fn issue<C>(&self, cmd: C) -> Result<Json, error::CmdError>
```

Comment:
Hmm, I thought I did this on purpose so that the returned future wouldn't be tied to the lifetime of the borrow of `&self`, but if everything compiles fine I suppose it doesn't matter any more.

Reply:
yeah I wasn't sure if I should leave it as is, but it did compile so I was like 🤷‍♂️ why not?